deployment/pypi/setup.py

```diff
@@ -63,7 +63,7 @@
         'scipy',
         'coverage',
         'colorama',
-        'scikit-learn>=0.20,<0.22',
+        'scikit-learn>=0.23',
```

Comment:
why this one is 0.23 and or others are 0.23.2?

Reply:
Both of they are ok. The 0.23.2 is more clear. Have modified.